### DISCUSSION TITLE
OpenAndroMaps theme when deactivating offline maps

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#360] [Q&A] [answered]
I have downloaded several OAM maps, one is activated and I use the Elevate theme with the subset Hiking. When I deactivate the map the theme changes to Biker automatically and the biker theme is stored in the current Profile without any action. When I activate a map again, the theme stays in Biker. It should stay all the time in the Elevate theme. Can you reproduce this problem?

**Comment by devemux86** [comment]
@kds112 thanks for the report.

To clarify, you deactivate all offline maps so the online map is activated?
Then you activate again an OpenAndroMaps map and the Elevate theme should be used.

I can reproduce it and will improve this workflow.

**Reply by kds112** [reply]
That is exactly what happened. Thanks for the quick reply 👍

**Comment by devemux86** [comment]
@kds112 please check the latest:
- https://github.com/devemux86/cruiser/discussions/329

Elevate or Voluntary theme for OpenAndroMaps maps and Freizeitkarte theme for Freizeitkarte maps
are now automatically activated when returning from online map (but with the default layers enabled).

**Reply by kds112** [reply]
It works very well 👍

**Comment by devemux86** [comment, marked as answer]
> Elevate or Voluntary theme for OpenAndroMaps maps and Freizeitkarte theme for Freizeitkarte maps
are now automatically activated when returning from online map (but with the default layers enabled).

Available in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
